File: src/accepted/more/uva10302.c
#include<stdio.h>

int main(){
int x, i;
double sum;
while(scanf("%d",&x) == 1){
sum = 0;
for(i=1;i<=x;i++)
sum += pow(i,3);
//printf("%.0f\n",sum);
printf("%d\n",x);
}
return 0;
}

File: src/vol103/uva10302.java
package vol103;

import java.util.Scanner;

class uva10302 {
   public static void main(String args[]){
       Scanner s = new Scanner(System.in);
       int x;
       long sum;
       while (s.hasNext()) {           
           x = s.nextInt();
           sum = 0;
           for (int i = 1; i <= x; i++) {
               sum += (long) Math.pow(i, 3);
           }
           System.out.printf("%d\n",sum);
       }
   }
}